packages/vite/src/node/__tests__/plugins/dynamicImportVar/parse.test.ts

```diff
@@ -7,8 +7,11 @@ const __dirname = resolve(fileURLToPath(import.meta.url), '..')
 
 async function run(input: string) {
   const { glob, rawPattern } =
```

Comment:
I think the test is ok because the e2e test is run well. may be it need to `normalizePath`?

Reply:
Thank you, yes, the importer needs to normalizePath.